en/use-auto-failover.md

```diff
@@ -48,76 +48,36 @@ Assume that there are 3 nodes in a PD cluster. If a PD node is down for over 5 m
+
+If **all** of the failed Pods have recovered, and you want to scale in the newly created Pods, you can follow the below procedure:
+
+Configure `spec.tikv.recoverFailover: true` (Supported since TiDB Operator v1.1.5):
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 kubectl edit tc -n ${namespace} ${cluster_name}
 ```
 
-```
-...
-status
-  tikv:
-    failureStores:
-      "1":
-        podName: cluster1-tikv-0
-        storeID: "1"
-      "2":
-        podName: cluster1-tikv-1
-        storeID: "2"
-...
-```
-
-After the `cluster1-tikv-0` node turns back to normal, you can delete it as shown below:
-
-```
-...
-status
-  tikv:
-    failureStores:
-      "2":
-        podName: cluster1-tikv-1
-        storeID: "2"
-...
-```
+TiDB Operator will scale in the newly created Pods automatically, when the scaling in is finished, configure `spec.tikv.recoverFailover: false` to avoid the auto-scaling operation when the next failover occurs and recovers.
 
 ### Failover with TiDB
 
 The TiDB automatic failover policy works the same way as `Deployment` does in Kubernetes. Assume that there are 3 nodes in a TiDB cluster. If a TiDB node is down for over 5 minutes (configurable by modifying `tidbFailoverPeriod`), TiDB Operator creates a new TiDB node. At this time, there are 4 nodes in the cluster. When the failed TiDB node gets back online, TiDB Operator deletes the newly created node and the number of nodes gets back to 3.
 
 ### Failover with TiFlash
 
-When a TiFlash Pod fails, its store status turns to `Disconnected`. After 30 minutes (configurable by modifying `max-store-down-time = "30m"` in the `[schedule]` section of the `pd.config` file), the store status turns to `Down`. After waiting for 5 minutes (configurable by modifying `tiflashFailoverPeriod`), TiDB Operator creates a new TiFlash Pod if this TiFlash Pod is still down. If the failed TiFlash Pod gets back online, TiDB Operator does not automatically delete the newly created Pod, and you need to manually drop it and restore the original number of Pods. To do this, you can delete the TiFlash Pod from the `status.tiflash.failureStores` field of the `TidbCluster` object.
+When a TiFlash Pod fails, its status turns to `Disconnected`. After 30 minutes (configurable by modifying [`max-store-down-time`](https://pingcap.com/docs/stable/pd-configuration-file/#max-store-down-time) in PD's [configuration file](https://github.com/pingcap/pd/blob/master/conf/config.toml)), it turns to `Down`. After waiting for 5 minutes (configurable by modifying `tiflashFailoverPeriod`), TiDB Operator creates a new TiFlash Pod if this TiFlash Pod is still down. If the failed TiFlash Pod gets back online, considering that scaling in the TiFlash Pods will trigger data transfer, TiDB Operator does not automatically delete the newly created Pod.
```

Comment:
```suggestion
When a TiFlash Pod fails, its status turns to `Disconnected`. After 30 minutes (configurable by setting [`max-store-down-time`](https://pingcap.com/docs/stable/pd-configuration-file/#max-store-down-time) to `"30m"` in PD's configuration file), the status becomes `Down`. After waiting for another 5 minutes (configurable by modifying `tiflashFailoverPeriod`), if this TiFlash Pod is still down, TiDB Operator creates a new TiFlash Pod.

If the failed TiFlash Pod gets back online, TiDB Operator does not automatically delete the newly created Pod. This is because scaling in the TiFlash Pods will trigger data transfer.
```